backend/YouTubeV2/Program.cs

```diff
@@ -44,6 +49,33 @@
             .WithExposedHeaders("Content-Disposition"));
 });
 
+builder.Services.Configure<DataProtectionTokenProviderOptions>(options =>
+       options.TokenLifespan = TimeSpan.FromHours(2));
+
+var jwtSettings = new JwtSettings(builder.Configuration.GetSection("JWTSettings"));
+//var jwtSettings = new JwtSettings();
+builder.Services.AddSingleton(jwtSettings);
+
+builder.Services.AddAuthentication(options =>
+{
+    options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+    options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+})
+.AddJwtBearer(options =>
+{
+    options.TokenValidationParameters = new TokenValidationParameters
+    {
+        ValidateIssuer = true,
+        ValidateAudience = true,
+        ValidateLifetime = true,
+        ValidateIssuerSigningKey = true,
+        ValidIssuer = jwtSettings.ValidIssuer,
+        ValidAudience = jwtSettings.ValidAudience,
+        IssuerSigningKey = new SymmetricSecurityKey(jwtSettings.SecurityKey)
+    };
+});
+builder.Services.AddScoped<JwtHandler>();
```

Comment:
move this so that all services are added in the same place. Maybe even create a function that adds all services